include/pluginlib/class_loader_imp.h

```diff
@@ -385,15 +385,15 @@ namespace pluginlib
   {
     //remove the package name to get the raw plugin name
     std::vector<std::string> split;
-    boost::split(split, lookup_name, boost::is_any_of("/"));
+    boost::split(split, lookup_name, boost::is_any_of("/:"));
```

Comment:
This is the core of the change.

The rest is my editor stripping out excess white space.